City-of-Pori/pori-kada
# PORI-375 Fixed style bugs caused by filtering hiding some of the item…

…s and showing others

To test:

1. drush cc all
2. Go to for example https://local.visitpori.fi/nae-ja-koe/aktiviteetit and filter the items. Then resize the browser and make sure there is always space between the items and that the layout looks nice.
